lib/openstack_query/query_executer.py

```diff
@@ -126,5 +126,6 @@ def run_query(
         logger.debug("run completed - time elapsed: %s seconds", time.time() - start)
 
         if self._parse_func:
-            results = self.parse_func(results)
+            parse_func = self._parse_func
+            results = parse_func(results)
```

Comment:
Nit, we can just `results = self._parse_func(results)`

Reply:
I thought that would fix it but apparently not

Reply:
Hmm, could we revert and possible suppress?
Usually though the warnings are pretty good and point at something real, if it's mis-typing or just param mis-matches

Reply:
I think its complaining because it thinks we're calling a method - but we're instead taking the output of the property and calling that since the output is a function

Reply:
Yeah looking at the warning:
`self.parse_func is not callable`

The typing just needs to be included

```
self.parse_func: Optional[Callable] = None
```

Reply:
yea I just used the internal attribute instead